src/Frontend/Components/ProgressBar/ProgressBar.util.tsx

```diff
@@ -232,33 +232,46 @@ function interpolateFromRedToGreen(
   return `hsl(${hue.toFixed(0)} 100 45)`;
 }
 
+function roundPercentagesToAtLeastOnePercentAndNormalize(
+  progressBarSteps: Array<ProgressBarStep>,
+): Array<ProgressBarStep> {
+  //percentages and steps need to be in parallel here
+  let percentages = progressBarSteps.map((step) => step.widthInPercent);
+  percentages = roundToAtLeastOnePercentAndNormalize(percentages);
```

Comment:
i don't think we need a mutable variable here:
```suggestion
  const percentages = roundToAtLeastOnePercentAndNormalize(progressBarSteps.map((step) => step.widthInPercent));
```